### PR TITLE
MAP-19 Reflect changes in emrapi metadata management

### PR DESCRIPTION
**Pull request by PawelGutkowski**
This PR depends on openmrs/openmrs-module-emrapi#98, please do not merge it until it is merged.

### CODE REVIEW

**Comment by dkayiwa** [comment]
@PawelGutkowski could you look into these conflicts? :)

**Comment by PawelGutkowski** [comment]
conflicts resolved

**Comment by dkayiwa** [comment]
@PawelGutkowski did you notice the travis failure?

**Comment by PawelGutkowski** [comment]
Changes in this PR depend on changes from PR on EMR-API https://github.com/openmrs/openmrs-module-emrapi/pull/98. As long as we don't merge it, builds of this PR will fail.

File: pom.xml [hunk 1]
```diff
@@ -68,6 +69,12 @@
     </dependencyManagement>
     
 		<dependencies>
+			<dependency>
+				<groupId>org.openmrs.web</groupId>
+				<artifactId>openmrs-web</artifactId>
+                <version>${openMRSVersion}</version>
+				<type>jar</type>
```
Comment (dkayiwa) [inline pom.xml]: Do you actually want to include this dependency in the compiled omod?

File: api/src/test/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivatorTest.java [hunk 2]
```diff
@@ -53,14 +60,6 @@ public boolean matches(Object argument) {
         }));
 
         verify(identifierSourceService).saveAutoGenerationOption(any(AutoGenerationOption.class));
-
-        verify(administrationService).saveGlobalProperty(argThat(new ArgumentMatcher<GlobalProperty>() {
-            @Override
-            public boolean matches(Object argument) {
-                GlobalProperty actual = (GlobalProperty) argument;
```
Comment (dkayiwa) [inline api/src/test/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivatorTest.java]: Do you have an alternative test for this?

**Comment by dkayiwa** [comment]
@PawelGutkowski did you see the followup comments i put on this pull request? :)

**Pull request merged**